### PR TITLE
fix some flaws in transaction test fillers  + EIP2028 fillers

### PR DESCRIPTION
**Pull request by holiman**
This PR addreses some quirks in the fillers, namely
- removes the use of arithmetic within json fields, by replacing all instances of e.g. `2^63-1` with the hex equivalent, 
- fixes some cases of shadowed fields, where e.g. `s` is present in two locations, both representing the secretkey and also the actual `s` value

### CODE REVIEW

**Comment by winsvega** [comment]
2**63-1 was intended to explain what the number means

**Comment by holiman** [comment]
> 2**63-1 was intended to explain what the number means

Yes, I realized that later, however, it's not proper json to have multiple keys in a dict. As it was, it was causiing errors in our parser. If you really want it there, it can be replaced with `"_comment"`or something ?

**Comment by winsvega** [comment]
In state tests we use '//' for comment fields. Such fields get removed and not exported to the compiled test.

**Comment by holiman** [comment]
I now added two fillers for EIP 2028. The geth-filled tests are availalbe at https://github.com/holiman/IstanbulTests/tree/master/TransactionTests/ttEIP2028 for now

File: src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "DataTestZeroBytes": {
```
Comment (shemnon) [inline src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json:2]:
```suggestion
  "DataTestInsufficientGas2028": {
```

File: src/TransactionTestsFiller/ttEIP2028/DataTestSufficientGas2028Filler.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "DataTestZeroBytes": {
```
Comment (shemnon) [inline src/TransactionTestsFiller/ttEIP2028/DataTestSufficientGas2028Filler.json]:
```suggestion
  "DataTestSufficientGas2028": {
```

File: src/TransactionTestsFiller/ttGasLimit/TransactionWithHihghGasLimit63m1Filler.json [hunk 3]
```diff
@@ -13,8 +13,7 @@
         "transaction" :
         {
             "data" : "",
-	         "gasLimit" : "2^63 - 1",
-            "gasLimit" : "9223372036854775807",
+	         "gasLimit" : "0x7fffffffffffffff",
```
Comment (shemnon) [inline src/TransactionTestsFiller/ttGasLimit/TransactionWithHihghGasLimit63m1Filler.json:16]:
```suggestion
            "gasLimit" : "0x7fffffffffffffff",
```

File: src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        ],
+        "result": "invalid"
+      }
+    ],
+    "transaction": {
+      "_comment1": "35 bytes, 4 z-bytes and 31 nz bytes. ",
+      "_comment2": "EIP 2028: 21000 + 4*4 + 31*16 = 21512 gas",
+      "_comment3": "Others  : 21000 + 4*4 + 31*68 = 23124 gas",
+      "data": "0xdeadbeef0000000101010010101010101010101010101aaabbbbbbcccccccddddddddd",
+      "_commet4": "gaslimit is set just below the intrinsic",
```
Comment (gumb0) [inline src/TransactionTestsFiller/ttEIP2028/DataTestInsufficientGas2028Filler.json]:
```suggestion
      "_comment4": "gaslimit is set just below the intrinsic",
```

**Comment by gumb0** [comment]
I tried filling these tx tests with testeth and got the same result, see https://github.com/ethereum/tests/pull/629

**Comment by winsvega** [comment]
moved to https://github.com/ethereum/tests/pull/630

**Pull request closed without merging**